includes/dashboard/showSelectedGroup.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/* 
+ * this is to show the selected group of the users, selected by the admin
+ */
+session_start();
+
+if( !( isset( $_SESSION['Username'] ) && isset($_SESSION['Name']) && $_SESSION['Username'] === 'admin' ) )
```

Comment:
__[code_sniffer]__
* Expected 1 space after IF keyword; 0 found
* Expected 0 spaces after opening bracket; 1 found
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 1

<!-- Comment by [SideCI](https://sideci.com) -->